### PR TITLE
Add more details to the message in the "Finalizing capture" dialog

### PR DESCRIPTION
**Pull request by dpallotti**
Bug: http://b/171350567

### CODE REVIEW

**Comment by dpallotti** [comment]
I wrote down something but feel free to suggest a different message.

File: OrbitQt/orbitmainwindow.cpp [hunk 1]
```diff
@@ -81,8 +81,17 @@ OrbitMainWindow::OrbitMainWindow(QApplication* a_App,
     setWindowTitle({});
   });
 
+  constexpr const char* kFinalizingCaptureMessage =
+      "<div align=\"left\">"
+      "Please wait while the capture is being finalized..."
+      "<ul>"
+      "<li>Waiting for the remaining capture data</li>"
+      "<li>Processing callstacks</li>"
+      "<li>Cleaning up dynamic instrumentation trampolines</li>"
```
Comment (ronaldfw) [inline OrbitQt/orbitmainwindow.cpp:90]: Maybe remove the word "trampolines"? Seems too much of a detail that users don't have to know about necessarily.

**Review by ronaldfw — APPROVED** [review]
LGTM, minor suggestion.

File: OrbitQt/orbitmainwindow.cpp (hunk 1 above)
Comment (dpallotti) [reply OrbitQt/orbitmainwindow.cpp:90]: I'm totally fine with it. So just "Cleaning up dynamic instrumentation"?

Comment (ronaldfw) [reply OrbitQt/orbitmainwindow.cpp]: SGTM

**Pull request merged**